docs/manualInstallation.md

```diff
@@ -0,0 +1,56 @@
+# Install karydia by hand
+It is possible to install karydia manually without the need of Helm and Tiller. This process is quite more complex and needs patience. If you just want to install karydia have a look at the [installation with Helm](../install/README.md).
```

Comment:
but Helm is still needed (locally) ... only the server part `Tiller` isn't needed anymore

Reply:
Removed the "without helm" part.